examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
+local xpath = require 'luaxpath'
+local cjson = require 'cjson'
+
+local _M = {}
+
+local initial_access_token = "CHANGE_ME_INITIAL_ACCESS_TOKEN"
```

Comment:
I don't want to make the client registrations code part of APIcast as I don't think it belongs in the API Gateway.

However, customers want to see a solution that synchronises clients in 3scale and RHSSO without needing additional work/infrastructure on their side

Adding this as an example is my compromise, but I don't want to add this into the main APIcast nginx.conf as an env variable. 

Happy to work with other suggestions, e.g @mikz suggestion for APIcast supporting custom env variables